packages/holocron/API.md

```diff
@@ -245,7 +189,7 @@ An action creator that loads Holocron modules and their data.
 ```js
 import { composeModules } from 'holocron';
 
-const load = (props) => (dispatch) => dispatch(composeModules([
+const loadModuleData = (props) => (dispatch) => dispatch(composeModules([
```

Comment:
This needs to be refactored as `loadModuleData` returns `void` not a `thunk`. 

```js
({ store: { dispatch }, ownProps }) => dispatch(composeModules( //...
```

Reply:
😅 thanks! that one escaped me... I should have done the generator PR before this one!

Reply:
fixed on https://github.com/americanexpress/holocron/pull/27/commits/db59438903e66e352ceabe081e6e3648a384bde9

Reply:
Actually `loadModuleData` returns `Promise<void>` technically my bad. Looks good